### ISSUE TITLE
[BUG] - Padding missing on Labs page for screens under 1300 px

### ISSUE THREAD

**Issue by trallard** [Quansight/Quansight-website#189]
### What site is this for?

Quansight Labs

### Expected behavior

There should be padding or margin before the text on the Home page regardless of the screen width. 

### Actual behavior

As soon as the width goes under 1300px this margin disappears, which means this applies to:
1. Desktop
2. Mobile (inc. tablets)

### How to Reproduce the problem?

1. Open either the preview or production Labs home page
2. Reduce the screen width below 1300px

### Anything else?

![Home_Page___Labs](https://user-images.githubusercontent.com/23552331/173381130-268ccb60-eb7f-4329-99ca-9144ecbbfc98.png)

**Comment by bskinn** [comment]
Removing this from LLC label since the `/labs` page has been removed from the LLC sitemap
